### PR TITLE
Signup signin

### PR DESCRIPTION
**Pull request by WriteCodeRAM**
Working on using authJS [universal auth](https://authjs.dev/guides/upgrade-to-v5) instead

### CODE REVIEW

File: src/middleware.ts
Comment (styled-man) [inline src/middleware.ts:1]: I don't think this is necessary

File: src/app/api/auth/[...nextauth]/options.ts
Comment (styled-man) [inline src/app/api/auth/[...nextauth]/options.ts:1]: Format this to get rid of the weird spacing

1. Right Click
2. Format Document

image

File: src/app/page.tsx [hunk 1]
```diff
@@ -4,4 +4,5 @@ import Image from "next/image";
 export default function Home() {
   // blank
   return <></>;
+
```
Comment (styled-man) [inline src/app/page.tsx:7]: I don't think this does anything

**Review by styled-man — CHANGES_REQUESTED** [review]

File: src/app/page.tsx (hunk 1 above)
Comment (WriteCodeRAM) [reply src/app/page.tsx]: Left it in there from main.

**Comment by 02danieljonas** [comment]
LGTM!

**Review by 02danieljonas — DISMISSED** [review]
LGTM!!

**Review by 02danieljonas — APPROVED** [review]

**Review by styled-man — APPROVED** [review]

**Review by styled-man — APPROVED** [review]

**Pull request merged**